src/libs/render-utils.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const cloudPlatforms = { gcp: 'Google Cloud Platform', azure: 'Microsoft Azure' };
+/**
+ * Render the cloud platform cell contents of a table
+ * @param row
+ * @returns {JSX.Element}
+ */
+export const renderCloudPlatforms = (row) => (
+  <div>
+    {Array.from(new Set(row.storage.map((s) => cloudPlatforms[s.cloudPlatform]))).map((c) => (
+      <div>{c}</div>
```

Comment:
you need a key on the div (any time you're working with arrays in React)

You'll probably notice the console warning

Reply:
key?

Reply:
Oh yeah, I think I ran into that error message.  Fix here - https://github.com/DataBiosphere/jade-data-repo-ui/pull/1057/files#diff-d2950d9d693e1a22b29406a1ec27e9905d0f794a113ca9b9f897f9dfb929996cR81

Reply:
Added